src/care/DefaultMacros.h

```diff
@@ -548,6 +548,15 @@
 #define CARE_CHECKED_PARALLEL_LOOP_END(CHECK) }); \
    CARE_NEST_END(CHECK) }}
 
+#define CARE_CHECKED_STREAMED_LOOP_START(RESOURCE, INDEX, START_INDEX, END_INDEX, CHECK) { \
```

Comment:
If you search for CARE_CHECKED_PARALLEL_LOOP_START, you'll see that it is defined one of two ways, depending on whether we have passed an option to use a legacy mode. You'll need to add another definition for CARE_CHECKED_STREAMED_LOOP_START/END in the legacy mode case.